artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -4000,6 +4000,23 @@
                </xsd:annotation>
             </xsd:element>
 
+            <xsd:element name="page-limit-bytes" type="xsd:string" default="-1" maxOccurs="1" minOccurs="0">
+               <xsd:annotation>
+                  <xsd:documentation>
+                     After the address entering into page mode, this attribute will configure how many pages can be written into page before activating the page-full-policy.
```

Comment:
This reads a bit funny, should it be "After the address enters into page mode...."  same for the below comment.